js/app/Data/TileBounds.js

```diff
@@ -113,20 +113,36 @@ define([
     var temporalExtents = args.temporalExtents || TileBounds.temporalExtents;
     var tilesPerScreen = args.tilesPerScreen;
 
-    var params = TileBounds.tileParamsForRange(args);
-    Logging.main.log("Data.BaseTiledFormat.tileBoundsForRange", params);
+    if (typeof(temporalExtents) == "number") {
+      var params = TileBounds.tileParamsForRange(args);
+      Logging.main.log("Data.BaseTiledFormat.tileBoundsForRange", params);
+
+      res = [];
+      for (var t = params.tilestart; t < params.tileend; t += temporalExtents) {
+        res.push(new Timerange([
+          new Date(t), new Date(t + temporalExtents)]));
+      }
 
-    res = [];
-    for (var t = params.tilestart; t < params.tileend; t += temporalExtents) {
-      res.push(new Timerange([
-        new Date(t), new Date(t + temporalExtents)]));
+      return {
+        set: res,
+        tilesPerScreen: Math.ceil(tilesPerScreen / Math.max(1, params.length / temporalExtents)),
+        params: params
+      };
+    } else {
+      /* temporalExtents is a list of time ranges: [[start,end],...] */
+
+      var range = new Timerange(args.bounds);
+
+      var res = temporalExtents.map(function (tileRange) {
+        return new Timerange([new Date(tileRange[0]), new Date(tileRange[1])]);
+      }).filter(function (tileRange) {
+        return range.intersectsObj(tileRange, {inclusive: false});
+      });
+      return {
+        set: res,
+        tilesPerScreen: Math.ceil(tilesPerScreen / Math.max(1, res.length))
+      };
```

Comment:
What would they contain? Params is used to calculate set and tilesPerScreen, and only returned for easier debugging. In this case we don't really have any params, because we have a hardcoded list instead...